### PR TITLE
Fix: Deprecated Connection::PARAM_*_ARRAY constants

### PR DESCRIPTION
**Pull request by blankse**
See https://github.com/pimcore/pimcore/pull/15843

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,8 +13,10 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "^11.0",
-    "pimcore/admin-ui-classic-bundle": "^1.1"
+    "php": "~8.1.0 || ~8.2.0",
```
Comment (kingjia90) [inline composer.json:16]: is this necessary?
```suggestion
```

**Review by kingjia90 — APPROVED** [review]
Rest LGTM, maybe we need to create a `CHANGELOG.md` and add info about the minimum requirement bump

File: composer.json (hunk 1 above)
Comment (blankse) [reply composer.json:16]: I think we should set supported php versions. But is a different topic yes.

Comment (kingjia90) [reply composer.json:16]: ok, but let's keep it in there, since i am also adding the "retroactive" changelog

**Pull request merged**